iq/iq_test.go

```diff
@@ -103,6 +103,26 @@ func TestAuditPackages(t *testing.T) {
 	assert.Equal(t, result, statusExpected)
 }
 
+func TestAuditPackagesIqCannotLocateApplicationID(t *testing.T) {
+	httpmock.Activate()
+	defer httpmock.DeactivateAndReset()
+
+	httpmock.RegisterResponder("GET", "http://sillyplace.com:8090/api/v2/applications?publicId=testapp",
+		httpmock.NewBytesResponder(200, []byte(`{ "applications": [] }`)))
+
+	var purls []string
+	purls = append(purls, "pkg:golang/github.com/go-yaml/yaml@v2.2.2")
+	purls = append(purls, "pkg:golang/golang.org/x/crypto@v0.0.0-20190308221718-c2843e01d9a2")
+
+	_, err := AuditPackages(purls, "testapp", setupIqConfiguration())
+	if err == nil {
+		t.Error("There is an error")
```

Comment:
Huh?? Did you mean  `There must be an error` or something like that ?

Reply:
Typo from the original PR, updated to something better in: 2b127ad